packages/lib/src/index.ts

```diff
@@ -42,10 +42,16 @@ export function isMessageOutput(output: Output): output is MessageOutput {
 }
 
 // CompletionOptions allows you to specify options for the completion request.
-export interface CompletionOptions<TModel> {
+export interface CompletionOptions<
+  TModel,
+  TInput extends Input = MessagesInput
```

Comment:
what will happen if i do:
```
const { text } = await windowAI.getCompletion({ prompt: "test" })
```
?

Will it work, or will it error because the completions object is assuming we're doing a message input?

Reply:
Yup just had to fix the output type as well: 
![image](https://user-images.githubusercontent.com/6723574/234408520-660ec669-4cc4-46d0-9c5c-b3fded317344.png)